lib/instana/activators/action_view.rb

```diff
@@ -12,8 +12,11 @@ def instrument
         require 'instana/instrumentation/action_view'
 
         ::ActionView::PartialRenderer
-          .prepend(Instana::Instrumentation::ActionView)
-
+          .prepend(Instana::Instrumentation::ActionView::PartialRenderer)
+        if ::Rails::VERSION::STRING > "6.0" && defined?(::ActionView::CollectionRenderer)
```

Comment:
I think if `::ActionView::CollectionRenderer` is defined, then we should prepend, regardless of the Rails version, afterall we have to cover all and any Rails versions.

Reply:
Yeah makes sense will remove rails version check!